bc_obps/reporting/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+class Report(models.Model):
+    title = models.CharField(max_length=100, db_comment="The title of the report")
+    description = models.TextField(db_comment="The description of the report")
+    created_at = models.DateTimeField(auto_now_add=True, db_comment="The timestamp when the report was created")
+
+    class Meta:
+        db_table_comment = "A table to store reports"
+        db_table = "erc.report"
```

Comment:
This line should be revised to `'erc"."report'` in order to correctly define the schema for this table. To test if it was added to the schema correctly, you can use `psql registration` and run `\d erc.report`.